modules/product/src/main/java/com/opengamma/strata/product/credit/type/StandardCdsConventions.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.credit.type;
+
+import static com.opengamma.strata.basics.currency.Currency.EUR;
+import static com.opengamma.strata.basics.currency.Currency.GBP;
+import static com.opengamma.strata.basics.currency.Currency.JPY;
+import static com.opengamma.strata.basics.currency.Currency.USD;
+import static com.opengamma.strata.basics.date.BusinessDayConventions.FOLLOWING;
+import static com.opengamma.strata.basics.date.DayCounts.ACT_360;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.EUTA;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.GBLO;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.JPTO;
+import static com.opengamma.strata.basics.date.HolidayCalendarIds.USNY;
+import static com.opengamma.strata.basics.schedule.Frequency.P3M;
+
+import com.opengamma.strata.basics.date.BusinessDayAdjustment;
+import com.opengamma.strata.basics.date.DaysAdjustment;
+import com.opengamma.strata.basics.date.HolidayCalendarId;
+
+/**
+ * Standardized credit default swap conventions.
+ */
+final class StandardCdsConventions {
+
+  private static final HolidayCalendarId GBLO_USNY = GBLO.combinedWith(USNY);
+  private static final HolidayCalendarId GBLO_USNY_JPTO = JPTO.combinedWith(GBLO_USNY);
+  private static final HolidayCalendarId GBLO_EUTA = GBLO.combinedWith(EUTA);
+
+  /**
+   * USD-dominated standardized credit default swap.
+   * <p>
+   * The payment dates are calculated with 'USNY'.
+   */
+  public static final ImmutableCdsConvention USD_STANDARD = ImmutableCdsConvention.of("USD-STANDARDIZED-CDS", USD, ACT_360,
```

Comment:
Name should be "USD-STANDARD", not "USD-STANDARDIZED-CDS". Similar change to other conventions